x-pack/test/functional/apps/machine_learning/anomaly_detection/anomaly_explorer.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+const JOB_CONFIG = {
```

Comment:
nit, it might be with explicitly supplying the type to the job and datafeed configs.
```
const JOB_CONFIG: Job = {
```

Reply:
Good idea, done in 2322004